CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file, per [the Keep a Changelog standard](http://keepachangelog.com/).
+
+## [Unreleased]
+
+## [1.1.0] - 2021-07-19
+### Added
+- Integration with [Co-Authors Plus](https://wordpress.org/plugins/co-authors-plus/). Props [@dinhtungdu](https://github.com/dinhtungdu) and [@Rahmon](https://github.com/Rahmon).
+
+### [1.0.0] - 2021-02-09
+
+Initial plugin release
+
+[Unreleased]: https://github.com/10up/ElasticPressLabs/compare/1.0.0...HEAD
```

Comment:
@Rahmon shouldn't this line be `https://github.com/10up/ElasticPressLabs/compare/1.1.0...HEAD`?